### PR TITLE
fix: dependabot updates and auto merges all @seamapi/* dependecies

### PR DESCRIPTION
**Pull request by kainpets**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -8,10 +8,3 @@ updates:
     versioning-strategy: "auto"
     allow:
       - dependency-name: "@seamapi/*"
-      - dependency-type: production
-    groups:
```
Comment (razor-x) [inline .github/dependabot.yml:12]: Can we still use a group so we get one pr for all the `@seamapi/*` deps?

Comment (kainpets) [reply .github/dependabot.yml:12]: This should work

File: .github/dependabot.yml [hunk 2]
```diff
@@ -8,10 +8,7 @@ updates:
     versioning-strategy: "auto"
     allow:
       - dependency-name: "@seamapi/*"
-      - dependency-type: production
     groups:
-      seam:
-        dependency-type: production
-        update-types:
```
Comment (razor-x) [inline .github/dependabot.yml:15]: Can we still limit to minor and patch? We should not auto merge major versions.

Comment (kainpets) [reply .github/dependabot.yml:15]: Only minor and patch updates to @seamapi/* production dependencies should be updated now

**Review by razor-x — APPROVED** [review]

**Pull request merged**